lib/services/models.js

```diff
@@ -5,7 +5,8 @@ const _ = require('lodash'),
   upgrade = require('./upgrade'),
   timer = require('../timer'),
   TIMEOUT_GET_COEFFICIENT = 2,
-  TIMEOUT_PUT_COEFFICIENT = 5;
+  TIMEOUT_PUT_COEFFICIENT = 5,
+  IN_NANOSECONDS = 1000000;
```

Comment:
why are we using nanoseconds ?  In clay-radio we're setting these properties as date objects which use milliseconds

Reply:
Well is the default unit for datadog, we could set it back to milliseconds and use a transform pipe I guess, but not sure if they can be applied for a specific log. 

https://docs.datadoghq.com/logs/explorer/facets/#units

Reply:
@luisbertelGL leave this as milliseconds as that's the JS standard. We modify it on ingestion into Datadog.